platforms/web/lib/dom.ts

```diff
@@ -16,18 +16,6 @@ limitations under the License.
 
 import { ComposerModel, DomHandle } from '../generated/wysiwyg';
 
-export function computeSelectionOffset(node: Node, offset?: number): number {
-    if (node && node.nodeType === Node.TEXT_NODE) {
-        return offset ?? node.textContent?.length ?? 0;
-    } else if (node.hasChildNodes()) {
-        return Array.from(node.childNodes)
-            .map((childNode) => computeSelectionOffset(childNode))
-            .reduce((prev, curr) => prev + curr, 0);
-    } else {
-        return 0;
-    }
-}
-
```

Comment:
Not sure about the history of this but this isn't used anywhere, so I deleted it. My guess is it was superseded by `computeNodeAndOffset` but never got removed.